test/unit/tasks/cms/cms_test.rb

```diff
@@ -2,24 +2,56 @@
 
 module Tasks
   class CmsTest < ActiveSupport::TestCase
+    class CmsTestFixPaginationHref < CmsTest
+      def setup
+        @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
+                                 published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
 
-    def setup
-      @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
-        published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
+        @page.save(validate: false)
+      end
 
-      @page.save(validate: false)
+      def test_fix_pagination_href
+        assert_equal true,  @page.published.include?('{{ part.link }}')
+        assert_equal false, @page.published.include?('{{ part.url }}')
+
+        execute_rake_task 'cms/cms.rake', 'cms:fix:pagination_href'
+
+        @page.reload
+
+        assert_equal true,  @page.published.include?('{{ part.url }}')
+        assert_equal false, @page.published.include?('{{ part.link }}')
+      end
     end
 
-    def test_fix_pagination_href
-      assert_equal true,  @page.published.include?('{{ part.link }}')
-      assert_equal false, @page.published.include?('{{ part.url }}')
+    class CmsTestFixSectionEmptyTitles < CmsTest
+      def setup
+        @provider = FactoryBot.create(:provider_account)
+      end
+
+      [nil, ""].each do |empty_val|
+        define_method("test_fix_section_empty_titles_title_#{empty_val.nil? ? 'nil' : 'empty'}") do
+          section = FactoryBot.build(:cms_section, partial_path: '/', parent: @provider.sections.root, title: empty_val, system_name: 'system-name')
+          section.save!(validate: false)
+          expected_title = section.id.to_s
+
+          execute_rake_task 'cms/cms.rake', 'cms:fix:section_empty_titles'
+
+          section.reload
+          assert_equal expected_title, section.title
```

Comment:
Maybe it would be better to assert title is _not_ empty? De we care what the title actually is?
```suggestion
          assert_not_empty expected_title
```

Reply:
Well, the rake task sets the title to something specific. I'd rather we check that it's actually set to the value we intended to, and not something potentially random.

Reply:
Agreed with @mayorova